irods/tests/irods_runtime_init.yml

```diff
@@ -37,6 +37,22 @@
       ansible.builtin.command: ils /testing/trash/home/shared
       changed_when: false
 
+    - name: Verify that the starting collections have UUIDs
+      ansible.builtin.shell:
+        executable: /bin/bash
+        cmd: |
+          set -o pipefail
+          iquest '%s' "select META_DATA_ATTR_VALUE where META_DATA_ATTR_NAME = 'UUID' and COLL_NAME = '{{ item }}'" | grep --quiet .
```

Comment:
This doesn't detect missing UUIDs. `iquest` writes "CAT_NO_ROWS_FOUND: Nothing was found matching your query" to stdout when it doesn't find anything, so `grep .` will always exit successfully.